src/harness/sas_connector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+
+__author__ = "Francisco Benavides (francisco.benavides@federatedwireless.com)"
+"""
+    SAS Connector
+    Federated Wireless
+
+    SAS to User Protocol Technical Specification
+    WINNF-16-S-0016 SAS to CBSD v0.3.10, 12 July 2016
+
+    9.2     Message Transport
+            HTTP v1.1, Content-type: application/json
+
+    SAS - SAS Interface Technical Specification
+    WINNF-16-S-0096 v0.3.5, 20 September 2016
+
+    7.2     Message Transport
+            HTTP 1.1, Content-type: application/json
+"""
+
+
+# Imports
+import os
+import logging
+
+
+# Constants
+
+# Log format
+LOG_FORMAT = logging.Formatter(
+    '%(asctime)-15s - SasConnector - '
+    '%(funcName)s - %(levelname)s - %(message)s')
+
+
+# Classes
+
+
+class SasConnector(object):
```

Comment:
I think we can also get rid of this class and put these function into another class or in config file.